src/index.html

```diff
@@ -272,33 +277,45 @@ <h4 class="card-title">
                     placeholder="Message to encrypt"
                     id="encryptMessageInput"
                   />
+                  <input
+                    class="form-control"
+                    type="text"
+                    placeholder="Recipient's Public Encryption Key"
+                    id="publicKeyInput"
+                  />
 
                   <button
                     class="btn btn-primary btn-lg btn-block mb-3"
                     id="encryptButton"
-                    disabled
```

Comment:
Similarly, this one was meant to be disabled until we have a public encryption key to encrypt with.